chapters/hpmor-chapter-008.tex

```diff
@@ -3,7 +3,7 @@ \chapter{Positive Bias}
 \begin{chapterOpeningAuthorNote}
 All these worlds are J.~K.~Rowling’s, except Europa. Attempt no fanfics there.
 
-One alert reviewer asked whether, if Luna is a seer, that means this is going to be an HPDM bottom!Draco mpreg fic. I regret that FFN does not allow me any larger font size in which to say NO. It honestly hadn’t occurred to me that Luna might be a \emph{real} seer—I'll have to decide whether to run with that or not—but I think we can all safely assume that if Luna \emph{is} a seer, she said something about “light planting a seed in darkness”, and Xenophilius, as always, interpreted this in rather the wrong way.
+One alert reviewer asked whether, if Luna is a seer, that means this is going to be an HPDM bottom!Draco mpreg fic. I regret that FFN does not allow me any larger font size in which to say NO. It honestly hadn’t occurred to me that Luna might be a \emph{real} seer—I'll have to decide whether to run with that or not—but I think we can all safely assume that if Luna \emph{is} a seer, she said something about “light planting a seed in darkness,” and Xenophilius, as always, interpreted this in rather the wrong way.
```

Comment:
This is a quotation ("she said something about"), and the comma isn't part of the quotation, so doesn't go inside the quotes.